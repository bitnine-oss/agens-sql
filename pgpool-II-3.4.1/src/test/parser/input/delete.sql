DELETE FROM t;
DELETE ;
DELETE FROM ONLY xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx;
DELETE FROM t WHERE x = 3;
DELETE FROM t WHERE x IN (select * from t1);
DELETE FROM foo WHERE a = 1 RETURNING *;
DELETE FROM foo WHERE a = 1 RETURNING a;
DELETE FROM foo WHERE a = 1 RETURNING a,b;
DELETE FROM foo WHERE CURRENT OF $1
DELETE FROM foo WHERE CURRENT OF cur
