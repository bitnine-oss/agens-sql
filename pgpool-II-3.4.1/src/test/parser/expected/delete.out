DELETE FROM "t"
syntax error: DELETE ;
DELETE FROM "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
DELETE FROM "t" WHERE  ("x"=3 )
DELETE FROM "t" WHERE "x" IN ( SELECT * FROM "t1")
DELETE FROM "foo" WHERE  ("a"=1 ) RETURNING *
DELETE FROM "foo" WHERE  ("a"=1 ) RETURNING "a"
DELETE FROM "foo" WHERE  ("a"=1 ) RETURNING "a","b"
syntax error: DELETE FROM foo WHERE CURRENT OF $1
DELETE FROM "foo" WHERE CURRENT OF cur
